pom.data.xml

```diff
@@ -284,6 +284,10 @@
                 <title>Azure Storage</title>
                 <packages>com.microsoft.azure.storage*</packages>
               </group>
+              <group>
```

Comment:
Update the version of this pom file as this module is newly introduced which warrants a new version. Customers using the current version shouldn't automatically see this inkrecognizer module.

Reply:
@srnagar I don't believe we need to bump the version for this pom for dev-time plugin configurations. It is only necessary for changes that are needed for runtime changes to the parents, for example the version properties for the dependencies.